Source/IGListSingleSectionController.m

```diff
@@ -72,6 +87,10 @@ - (UICollectionViewCell *)cellForItemAtIndex:(NSInteger)index {
                                                           bundle:self.bundle
                                             forSectionController:self
                                                          atIndex:index];
+    } else if ([self.identifier length] > 0) {
```

Comment:
This is fine for now, but we probably want to turn this into a state enum and avoid too many ifs. Don't worry about it here, just noting to myself.